Source/Core/buildModuleUrl.js

```diff
@@ -5,6 +5,7 @@ import Resource from "./Resource.js";
 
 /*global CESIUM_BASE_URL,define,require*/
 
+// TODO: all these should have JSDoc tags, or mark as private
```

Comment:
To get this merged, there shouldn't be any `TODO` comments in the code. Any action item should be in an issue so we can track them. Or let me know if there was discussion around any of these that wasn't resolved.

Reply:
I took out the comment, should be addressed by https://github.com/CesiumGS/cesium/issues/8922 which you already re-opened.